src/config/rate-limiting-error.ts

```diff
@@ -0,0 +1,12 @@
+export class RateLimitingError extends Error {
```

Comment:
This code is as is, it was in `enhance-octokit.ts` just split it out into seperate file.
Should be able to remove it soon, couple of tests rely on it so will do it seperately.